test/test_audio.cpp

```diff
@@ -0,0 +1,32 @@
+
+
+#include <gtest/gtest.h>
+
+#include "Audio.h"
+
+TEST(Audio, initialisationIsAllocation) {
```

Comment:
for some reason the initialization in line 9 fails, if i put the :
` constexpr int numberOfSamples = 600000;`
then the test passes, also i think in line 10 and 24
`for (int i = 0; i < numberOfSamples; ++i) rawAudio[i] = i % INT16_MAX;`

Reply:
Yes, I left it to be, looks like there is a problem with the size of the std:array , but we are not going to need arrays this large ever, because array here is for short audio frames.